Resources/Prototypes/_Impstation/Entities/Mobs/Customization/Markings/headcovers.yml

```
@@ -0,0 +1,149 @@
+#regrettably due to technical limitations spiders cannot materially dress for religious purposes
+#TO DO - why is kippah white not works?
+
+#human,dwarf,slime
+- type: marking
+  id: BurqaSolid
+  bodyPart: Hair
+  markingCategory: Hair
+  speciesRestriction: [Human, Dwarf, SlimePerson, Oni, Goblin, Felinid] # Frontier
```

```suggestion
  speciesRestriction: [Human, Dwarf, SlimePerson, Oni, Goblin, Felinid] # Frontier: add Oni, Goblin, Felinid
```
Describe the changes.